packages/pages/src/vite-plugin/build/buildStart/rendering/wrapper.ts

```diff
@@ -39,9 +39,16 @@ export const reactWrapper = async <T extends TemplateRenderProps>(
 
   let clientHydrationString;
   if (hydrate) {
+    const templatePath =
+      Object.keys(manifest.clientPaths).length === 0
```

Comment:
When would they not exist?

Reply:
Hmmmm really only in tests, I can remove this and adjust tests